USER_GUIDE.md

```diff
@@ -53,6 +53,24 @@ If `comments` is given in the `features` list then this enables all commenting f
 
 > Note: All commands can be given with a prefix of either `Derek <command>` or `/<command>`.
 
+### Feature: `messages`
+
+Boilerplate messages can be applied for the time you would like to point someone to the contributing guide for example. You can just write `Derek message: docs` or `/message: docs` you can also use the short style `msg` instead of `message`. Then the content of that message is displayed.
```

Comment:
'Have Derek add pre-configured comments to a PR or Issue thread, for example when you would like to direct someone towards the contributing guide'

Reply:
Swapped the explanation 👍